apps/server/src/modules/tldraw/tldraw.module.ts

```diff
@@ -33,7 +32,7 @@ const defaultMikroOrmOptions: MikroOrmModuleSyncOptions = {
 		}),
 		ConfigModule.forRoot(createConfigModuleOptions(config)),
 	],
-	providers: [TldrawService, TldrawBoardRepo, TldrawRepo, YMongodb],
+	providers: [TldrawService, TldrawBoardRepo, TldrawRepo, YMongodb, XApiKeyStrategy],
 	controllers: [TldrawController],
 })
 export class TldrawModule {}
```

Comment:
can you add a test that checks that the authentication still works? I.e. for non authenticated user / client (no API_KEY), it should give a 403